qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -84,11 +84,13 @@ def __init__(
 
         Raises:
             ValueError:
-                - if ``quantum_kernel`` is passed and ``precomputed`` is set to ``True``. To use
-                a precomputed kernel, ``quantum_kernel`` has to be of the ``None`` type.
+                - if :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel` is passed and
```

Comment:
I think this change to make things specific makes it incorrect. The `quantum_kernel` parameter must be None if you say you will use a `precomputed` kernel when that parameter is True. A quantum_kernel by type just has to be subclass of BaseKernel - so by making some specific reference it changes the meaning somewhat.

Now its true that if precomputed is False and you do not pass a quantum_kernel, i.e. you leave it at None, then it creates an instance of that specific subclass - but overall its not limited to that. Arguably this default when None and precomputed is False ought to be noted in the docstring above.